SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -1,65 +1,68 @@
 <!doctype html>
 <html lang="en" ng-app="SECP">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>SECP</title>
+        <link rel="shortcut icon" type="image/ico" href="img/favicon.ico"/>
+        <link href="lib/bootstrap-css/css/bootstrap.css" rel="stylesheet">
+        <link href="lib/jquery-ui/themes/smoothness/jquery-ui.css" rel="stylesheet"/>
+        <link href="lib/components-font-awesome/css/font-awesome.min.css" rel="stylesheet"/>
+  	    <link href="lib/simple-line-icons/css/simple-line-icons.css" rel="stylesheet">
+ 	    <link href="img/home/device-mockups/device-mockups.css" rel="stylesheet">
+ 	    <link href="lib/simple-line-icons/css/simple-line-icons.css" rel="stylesheet">
+	    <link href="img/home/device-mockups/device-mockups.css" rel="stylesheet">
+        <link href="css/app.css" rel="stylesheet">
+        <link href="lib/sweetalert2/dist/sweetalert2.css" rel="stylesheet">
+    </head>
+    <body>
+        <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="/#/">SECP</a>
+                </div>
+                <!-- Collect the nav links, forms, and other content for toggling -->
+                <div class="collapse navbar-collapse navbar-ex1-collapse">
+                    <ul class="nav navbar-nav">
+                        <li>
+                            <a href="/#/login">Login</a>
```

Comment:
I changed this in my latest PR. I got rid of the hash-tags.

Reply:
I will change it now